packages/react/src/analytics/integrations/GTM/eventsMapper.ts

```diff
@@ -23,40 +107,64 @@ const eventsMapper: EventMappers = {
       products: get(properties, 'products', []).map(getProductData),
       list: properties.list,
       currency: properties.currency,
+      from: properties.from,
+      error: properties.error,
+      itemListId: properties.listId,
+      itemListName: properties.list,
```

Comment:
We have `list` and `itemListName` being mapped to the same property. Can't we use just one of them?

Reply:
Fixed! 👍🏼